cranelift/codegen/src/machinst/buffer.rs

```diff
@@ -406,7 +406,7 @@ impl<I: VCodeInst> MachBuffer<I> {
 
     /// Add a byte.
     pub fn put1(&mut self, value: u8) {
-        trace!("MachBuffer: put byte @ {}: {:x}", self.cur_offset(), value);
+        // trace!("MachBuffer: put byte @ {}: {:x}", self.cur_offset(), value);
```

Comment:
Commented-out lines here -- we should restore these or else delete them if we decide we don't need them (here and below).